src/Tribe/Editor/Meta.php

```diff
@@ -298,14 +373,30 @@ public function update_tickets_list_in_rest( $check, $unused_object_id, $meta_ke
 	 *
 	 * @since 4.10.11.1
 	 *
-	 * @return array
+	 * @return array List of ghost meta fields.
 	 */
 	public function get_ghost_meta_fields() {
 		return [
-			'_tribe_tickets_list',
-			'_tribe_ticket_going_count',
-			'_tribe_ticket_not_going_count',
-			'_tribe_ticket_has_attendee_info_fields',
+			'_tribe_tickets_list'                    => 1,
+			'_tribe_ticket_going_count'              => 1,
+			'_tribe_ticket_not_going_count'          => 1,
+			'_tribe_ticket_has_attendee_info_fields' => 1,
+		];
+	}
+
+	/**
+	 * Get global stock meta fields that we don't actually want to update/delete from normal WP REST routes.
+	 *
+	 * @since TBD
+	 *
+	 * @return array List of global stock meta fields.
+	 */
+	public function get_global_stock_meta_fields() {
+		return [
+			Tribe__Tickets__Global_Stock::GLOBAL_STOCK_ENABLED => 1,
+			Tribe__Tickets__Global_Stock::GLOBAL_STOCK_LEVEL   => 1,
+			Tribe__Tickets__Global_Stock::TICKET_STOCK_MODE    => 1,
```

Comment:
:warning: Expected 1 space between "TICKET_STOCK_MODE" and double arrow; 4 found.

<sub>hash: 767af852fc64a94469a5f02168b7a75c</sub>